diff --git a/lib/index.js b/lib/index.js
index 865e2c8029574cd39905881065582f7071bbaea9..13546ea148f8fabcd4ad6d6b34f0d705174647f0 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -29,7 +29,7 @@ class Reader {
         this.load(db);
     }
     load(db) {
-        if (!Buffer.isBuffer(db)) {
+        if (!require("buffer/").Buffer.isBuffer(db)) {
             throw new Error(`mmdb-lib expects an instance of Buffer, got: ${typeof db}`);
         }
         this.db = db;
diff --git a/lib/metadata.js b/lib/metadata.js
index 76a96e99c93fe802c7c941853377dff4415b3e29..f96b086419983285f0b1383abf46b4add458c0fc 100644
--- a/lib/metadata.js
+++ b/lib/metadata.js
@@ -6,7 +6,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.isLegacyFormat = exports.parseMetadata = void 0;
 const decoder_1 = __importDefault(require("./decoder"));
 const utils_1 = __importDefault(require("./utils"));
-const METADATA_START_MARKER = Buffer.from('ABCDEF4D61784D696E642E636F6D', 'hex');
+const METADATA_START_MARKER = require("buffer/").Buffer.from('ABCDEF4D61784D696E642E636F6D', 'hex');
 const parseMetadata = (db) => {
     const offset = findStart(db);
     const decoder = new decoder_1.default(db, offset);
