diff --git a/dist/utils/inject-script.mjs b/dist/utils/inject-script.mjs
index ef5aae321461f505e451a81fd43823ed02ab3d0e..a7e1992f19c0526a8fb8360a155f5969f8235597 100644
--- a/dist/utils/inject-script.mjs
+++ b/dist/utils/inject-script.mjs
@@ -15,7 +15,7 @@ import { browser } from "wxt/browser";
 async function injectScript(path, options) {
 	const url = browser.runtime.getURL(path);
 	const script = document.createElement("script");
-	if (browser.runtime.getManifest().manifest_version === 2) script.text = await fetch(url).then((res) => res.text());
+	if (browser.runtime.getManifest().manifest_version === 2 && false) script.text = await fetch(url).then((res) => res.text());
 	else script.src = url;
 	const loadedPromise = makeLoadedPromise(script);
 	await options?.modifyScript?.(script);
